runtime/assembly/__tests__/inherent.spec.ts

```diff
@@ -1,14 +1,13 @@
-import { Inherent } from '../models';
+import { InherentData } from '../models';
 import { MockBuilder } from './mock-builder';
 import { MockResult } from './mock-result';
 
 describe("Inherent", () => {
     it("Should instanciate new Inherent class from the empty SCALE Encoded Array", () => {
```

Comment:
We could change the name of the test to be ".... InherentData class ... " instead of `Inherent` class